lib/History/History.js

```diff
@@ -100,6 +74,14 @@ var History = module.exports = kind.singleton({
 		this._popStateHandler = this.bindSafely(this.popStateHandler);
 
 		this.enableBackHistoryAPIChanged();
+
+		this._appInfoReq = new Ajax({url:"appinfo.json"});
+		this._appInfoReq.response(function(inSender, inResponse) {
+			if(inResponse !== undefined) {
+				this.set("enableBackHistoryAPI", !inResponse.disableBackHistoryAPI);
+			}
+		}, this);
+		this._appInfoReq.go({});
```

Comment:
No need for the empty object `{}` here. Beyond the unnecessary memory allocation, it also forces enyo/Ajax to do more work.